### DISCUSSION TITLE
Power consumption

### DISCUSSION THREAD

**Discussion by jonte964** [astrandb/senz_hass#23] [Ideas]
Hi,

Would it be possible to add power consumption? 

With knowledge on when the relay is on and the size of the heat sling it would be possible.'

BR/J

**Comment by astrandb** [comment]
Yes, in theory. But it needs some work.
I stopped developing (and using) this version when the official integration was introduced in HA and I think further  development should be done in the official integration.

It should be fairly easy to follow relay status and display it in the climate card. However, I foresee challenges to monitor the time that the relay is closed in a stable way.
